.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,107 @@
+on:
+  push:
+    branches:
+      - main
+      - ci
+  pull_request:
+    branches:
+      - main
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macOS-latest,   r: 'devel',   vdiffr: false}
+          - {os: macOS-latest,   r: 'release', vdiffr: true}
+          - {os: windows-latest, r: 'release', vdiffr: true}
+          - {os: ubuntu-16.04,   r: 'release', vdiffr: true,  rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: 'oldrel',  vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.5',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.4',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.3',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.054,  r: '3.2',     vdiffr: false, rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
```

Comment:
Is this supposed to be 16.054, or 16.04 like the others? I noticed this build is failing with not much info, so I'm wondering if this is a typo and 16.054 is not an ubuntu version that github actions can find?

Reply:
Ah, I must have inputted an extra character when I copied it from your `sagethemes` check file, thank you! I was wondering why it was failing. I'll fix the commit I just made that removes it.